docs/explanations/how_to_interpret_the_security_status_command.md

```diff
@@ -0,0 +1,100 @@
+  version: 1:1.2.8.dfsg-2ubuntu4.3+esm1
+```
+
+Let's understand what each key mean on the output of the `ua security-status` command:
+
+* **summary**: The summary of the system related to Ubuntu Advantage (UA) and
+  the different package sources in the system:
+
+  * **num_esm_apps_packages**: The number of packages installed from `esm-apps`.
+  * **num_esm_apps_updates**: The number of `esm-apps` package updates available in the system.
+  * **num_esm_infra_packages**:  The number of packages installed from `esm-infra`.
+  * **num_esm_infra_updates**: The number of `esm-infra` package updates available in the system.
+  * **num_installed_packages**: The number of installed packages in the system.
+  * **num_main_packages**: The number of packages installed from the `main` archive component.
+  * **num_multiverse_packages**: The number of packages installed from the `multiverse` archive
+    component.
+  * **num_restricted_packages**: The number of packages installed from the `restricted` archive
+    component.
+  * **num_third_party_packages**: The number of packages installed from `third party` sources.
+  * **num_universe_packages**: The number of packages installed from the `universe` archive
+    component.
+  * **num_unknown_packages**: The number of packages installed from sources not known to `apt`
+    (installed locally through dpkg or packages without a remote reference).
+  * **num_standard_security_updates**: The number of standard security updates available in the system.
+
+  It is worth mentioning here that the updates count are verifying the number of **security**
+  updates for **installed** packages. For example, let's assume your machine has a universe package that
+  has a security update from `esm-infra`. The count will be displayed as:
+
+  ```
+  num_esm_infra_packages: 0
+  num_esm_infra_updates: 1
+  num_universe_packages: 1
+  ```
+
+  After upgrading the system, the count will turn to:
+
+  ```
+  num_esm_infra_packages: 1
+  num_esm_infra_updates: 0
+  num_universe_packages: 0
+  ```
+
+  * **ua**: An object representing the state of UA on the system:
+    * **attached**: If the system is attached to a UA subscription.
+    * **enabled_services**: A list of services that are enabled on the system. If unattached, this
+      will always be an empty list.
+    * **entitled_services**: A list of services that are entitled on your UA subscription. If
+      unattached, this will always be an empty list.
+
+* **packages**: A list of updates for security packages installed in the system.
+  Every entry on the list will follow this structure:
+
+  * **origin**: The host were the update comes from.
+  * **package**: The name of the package.
+  * **service_name**: The service that provides that package update. It can be either: `esm-infra`,
+    `esm-apps` or `standard-security`.
+  * **status**: The status for this update. It will be one of:
+    * **upgrade_available**: The package can be upgraded right now.
```

Comment:
Since these are string values, I think we should include quotes too
```suggestion
    * **"upgrade_available"**: The package can be upgraded right now.
```
and maybe with backticks too?
```suggestion
    * **`"upgrade_available"`**: The package can be upgraded right now.
```